src/test/features/response/routes/confirmation.ts

```diff
@@ -50,6 +53,24 @@ describe('Defendant response: confirmation page', () => {
           .set('Cookie', `${cookieName}=ABC`)
           .expect(res => expect(res).to.be.serverError.withText('Error'))
       })
+
+      it('should render states paid with less than claim amount with next step - NO MEDIATION', async () => {
+        claimStoreServiceMock.resolveRetrieveClaimByExternalId(fullDefenceWithStatesPaidLessThanClaimAmount)
+
+        await request(app)
+          .get(ResponsePaths.confirmationPage.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId }))
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.successful.withText(`The court will review the case. You might have to go to a hearing.`))
+      })
+
+      it('should render states paid with less than claim amount with next step - WITH MEDIATION', async () => {
+        claimStoreServiceMock.resolveRetrieveClaimByExternalId(fullDefenceWithStatesPaidLessThanClaimAmountWithMediation)
+
+        await request(app)
+          .get(ResponsePaths.confirmationPage.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId }))
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.successful.withText(`We’ll ask if they want to try mediation. If they agree, we’ll contact you with an appointment.`))
```

Comment:
is that a backtick instead of single quote? if it's not needed, i would use single quote.

Reply:
Fixed it now @KiranVarma02 . Thanks